### PR TITLE
fix(formatting): Fix single statement formatting for If statements - Fix warnings in InternalClient

### PR DESCRIPTION
**Pull request by azabbasi**
I have fixed some warnings in InternalClient
converted all If statement without braces to have braces.

### CODE REVIEW

File: iothub/device/tests/ModuleClientTests.cs [hunk 1]
```diff
@@ -174,7 +174,7 @@ await moduleClient
                     "custom data")
                 .ConfigureAwait(false);
 
-            await moduleClient.InternalClient.OnReceiveEventMessageCalled(null, null).ConfigureAwait(false);
+            await moduleClient.InternalClient.OnReceiveEventMessageCalledAsync(null, null).ConfigureAwait(false);
```
Comment (azabbasi) [inline iothub/device/tests/ModuleClientTests.cs:177]: This is a change I made that is irrelevant to the if statements.

File: common/src/Logging.Common.cs [hunk 2]
```diff
@@ -524,32 +571,32 @@ private unsafe void WriteEvent(int eventId, string arg1, string arg2, string arg
         {
             if (IsEnabled())
             {
-                if (arg1 == null) arg1 = "";
-                if (arg2 == null) arg2 = "";
-                if (arg3 == null) arg3 = "";
-                if (arg4 == null) arg4 = "";
+                arg1 ??= "";
```
Comment (azabbasi) [inline common/src/Logging.Common.cs:574]: took a page from Dave's book :)

**Comment by bikamani** [comment]
> I have fixed some warnings in InternalClient

Build/compilation warnings? or something else?

**Comment by azabbasi** [comment]
> > I have fixed some warnings in InternalClient
> 
> Build/compilation warnings? or something else?

code analysis warnings. When you look at a file in VS you will see `...` at the beginning of some lines. that means the statement either can be simplified or there is a useful suggestion for you to make.

File: iothub/device/src/InternalClient.cs [hunk 3]
```diff
@@ -259,12 +272,7 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
                 }
             }
 
-            if (!isFound)
-            {
-                return default(T);
-            }
-
-            return (T)handler;
+            return !isFound ? default : (T)handler;
```
Comment (abhipsaMisra) [inline iothub/device/src/InternalClient.cs:275]: ahh, I like the ternary operator! :(

File: iothub/device/src/InternalClient.cs [hunk 4]
```diff
@@ -600,12 +605,7 @@ public Task RejectAsync(string lockToken, CancellationToken cancellationToken)
         /// <returns>The lock identifier for the previously received message</returns>
         public Task RejectAsync(Message message)
         {
-            if (message == null)
-            {
-                throw new ArgumentNullException(nameof(message));
-            }
-
-            return RejectAsync(message.LockToken);
+            return message == null ? throw new ArgumentNullException(nameof(message)) : RejectAsync(message.LockToken);
```
Comment (abhipsaMisra) [inline iothub/device/src/InternalClient.cs:608]: nit: multiline here?

File: iothub/device/tests/DeviceClientTests.cs [hunk 5]
```diff
@@ -172,7 +172,7 @@ public async Task DeviceClient_OnMethodCalled_NullMethodRequest()
                 return Task.FromResult(new MethodResponse(Encoding.UTF8.GetBytes("{\"name\":\"ABC\"}"), 200));
             }, "custom data").ConfigureAwait(false);
 
-            await deviceClient.InternalClient.OnMethodCalled(null).ConfigureAwait(false);
+            await deviceClient.InternalClient.OnMethodCalledAsync(null).ConfigureAwait(false);
```
Comment (abhipsaMisra) [inline iothub/device/tests/DeviceClientTests.cs:175]: I had these in my PR as well. Once this goes in, it'll decrease the size of my PR. I like that!

**Review by abhipsaMisra — APPROVED** [review]
Looks good!
Could you share the regex you used for this find-replace, I was super impressed by the ease with which you made this change!

File: iothub/device/src/InternalClient.cs (hunk 4 above)
Comment (azabbasi) [reply iothub/device/src/InternalClient.cs:608]: I will make a note and update it in another PR.

**Pull request merged**